### DISCUSSION TITLE
How can I contribute to the documentation?

### DISCUSSION THREAD

**Discussion by maralihart** [microsoft/fluentui#26553] [Q&A] [answered]
How can I contribute to the documentation for Fluent UI v9? I'm a Microsoft employee if that makes any difference.

**Comment by layershifter** [comment, marked as answer]
@maralihart

- Pages specific to v9 are located there, https://github.com/microsoft/fluentui/tree/master/apps/public-docsite-v9/src
- Documentation for components is collocated to their package, for example https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-image/stories/Image
